src/webchat-ui/components/presentational/Badge.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+interface IBadgeProps {
+    content: number;
+    backgroundColor: string;
+    fontColor: string;
+}
+
+const Badge = (props: IBadgeProps) => {
+
+    const { content, backgroundColor, fontColor } = props;
+
+    const size = 24;
+
+    if (content === 0)
+        return null;
+
+    return (
+            <span style={{
```

Comment:
Inline styles make it hard to customise. Also should provide a stable class name string for customisations.